mllib/src/main/scala/org/apache/spark/ml/feature/QuantileDiscretizer.scala

```diff
@@ -49,6 +49,21 @@ private[feature] trait QuantileDiscretizerBase extends Params
 
   /** @group getParam */
   def getNumBuckets: Int = getOrDefault(numBuckets)
+
+  /**
+   * Relative error (see approxQuantile documentation for description).  Must be >= 0.
+   * By default, the relative error will be in the range [0.001, 0.01]. The exact value
+   * is computed based on the number of buckets, decreasing as the number of buckets
+   * increases. Explicitly setting the relative error will override this default calculation.
+   * @group param
```

Comment:
Provide a `@see` link to `approxQuantile` (https://github.com/apache/spark/blob/13ce10e95401b21fa40ca0bb27ebf9a0bfffe70f/sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala#L66).